src/component/handlers/edit/editOnPaste.js

```diff
@@ -176,4 +186,21 @@ function insertFragment(
   );
 }
 
+function areTextBlocksAndClipboardEqual(
+  textBlocks: Array<string>,
+  blockMap: BlockMap
+): boolean {
+  if (textBlocks.length != blockMap.size) {
+    return false;
+  }
+  let i = 0;
+  for (const block of blockMap) {
+    if (block[1].getText() !== textBlocks[i]) {
+      return false;
+    }
+    i++;
+  }
+  return true;
```

Comment:
Thanks! I'm pretty new to `immutable-js` so I missed that. I'll make the change and do some testing before submitting.